src/mustache_core.js

```diff
@@ -225,6 +225,13 @@ var core = {
 							return localPartialName(partialScope, {}, nodeList);
 						} else {
 							var domRenderer = core.getTemplateById(localPartialName);
+							//!steal-remove-start
+							if (process.env.NODE_ENV !== 'production') {
+								if (!domRenderer) {
+									dev.warn('Unable to find the partial "' + localPartialName + '" in ' + state.filename + ':' + state.lineNo);
```

Comment:
can you add the filename and lineNumber?

Reply:
@phillipskevin Done!